wiki/Ranking_Criteria/id.md

```diff
@@ -0,0 +1,267 @@
+# Kriteria ranking
+
+*Untuk kriteria ranking spesifik [game mode](/wiki/Game_mode), lihat: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), dan [osu!mania](osu!mania)*
+
+Kumpulan **kriteria ranking** ini menjabarkan [aturan dan pedoman](#general-terms) yang harus diikuti [beatmaps](/wiki/Beatmaps) untuk melalui [prosedur ranking beatmap](/wiki/Beatmap_ranking_procedure).
+
+Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum Ranking Criteria](https://osu.ppy.sh/community/forums/87). Lihat *[How to propose Ranking Criteria changes](https://osu.ppy.sh/community/forums/topics/720532)* untuk panduan kontribusi. Catat bahwa perubahan pada bahasa, tata bahasa, atau organisasi dokumen ini dapat melewati tahap post forum selama tidak mengubah arti aturan atau pedoman.
```

Comment:
```suggestion
Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum Kriteria Ranking](https://osu.ppy.sh/community/forums/87). Lihat *[How to propose Ranking Criteria changes](https://osu.ppy.sh/community/forums/topics/720532)* untuk panduan kontribusi. Catat bahwa perubahan pada bahasa, tata bahasa, atau organisasi dokumen ini dapat melewati tahap post forum selama tidak mengubah arti aturan atau pedoman.
```
Also for `How to propose Ranking Criteria changes`, umm... how about to change that link to `Bagaimana mengusulkan perubahan terhadap Kriteria Peringkat`? same as `forum Kriteria Ranking`.

`Lihat *[Bagaimana mengusulkan perubahan terhadap Kriteria Peringkat](https://osu.ppy.sh/community/forums/topics/720532)* untuk panduan kontribusi.`

Or, if you want to keep this link, i think it would be better if you keep `Ranking Criteria forum` from the link above.

Reply:
forum post titles are not translated, because...its odd to translate something untranslatable upon clicking.